src/backend/marsha/core/templates/core/lti_development.html

```diff
@@ -122,6 +122,7 @@ <h2>Iframe LTI Resource setup</h2>
         cross-origin call.
       </p>
       <form
+        id="lti_resource_iframe"
```

Comment:
Because it's the only view available in marsha for now, and we can only launch browser with a GET request.
Besides, it is how a user would reach those views.

Reply:
There is also the LTI view. It's just that it expects a POST with LTI parameters but you should be able to forge a request with your automated browsers knowing the LTI passport and post it?

Reply:
I don't get why it would be better trying to do that, as the LTI development view already contains the related forms.
The real test would be to call marsha through a LMS, though.

Reply:
The only interest I see is to really do an end-to-end test of what is used in production... There may not be a big difference between the development and the LTI view... but one may still be broken when the other is working.

Going through an LMS is not as important imo since the interface follows the LTI standard?

Reply:
Sorry, but I don't see how a forged post request would be closer to a real-world usage.

Reply:
Because when you forge the request you are doing what the LMS does, not something that Marsha is doing and that you would want to test here.

Then when you send your forged request to Marsha to the real LTI view used in production, you are testing the service exactly the way it is used in production... so it is the only way to be know if it works. For example, the development view is bypassing all LTI checks so anything broken in the LTI class would not be seen :man_shrugging:

Reply:
OK, I understand your fear now.
We use this view, but LTI verification is enabled, that's why I added oauth fields there.